src/NodaTime.Tools.Common/FileUtility.cs

```diff
@@ -10,24 +10,19 @@ namespace NodaTime.Tools.Common
 {
     public static class FileUtility
     {
-        public static byte[] LoadFileOrUrl(string source)
+        public static async Task<byte[]> LoadFileOrUrlAsync(string source)
         {
             if (source.StartsWith("http://") || source.StartsWith("https://") || source.StartsWith("ftp://"))
             {
-                // This is an ugly way of avoiding asynchrony, but we're only using it in
-                // command line tools.
-                var task = Task.Run(() => FetchContentAsync(source));
-                return task.Result;
+                return await FetchContentAsync(source);
             }
             return File.ReadAllBytes(source);
 
             async Task<byte[]> FetchContentAsync(string url)
             {
-                using (var client = new HttpClient())
-                {
-                    var response = await client.GetAsync(source);
-                    return await response.EnsureSuccessStatusCode().Content.ReadAsByteArrayAsync();
-                }
+                using var client = new HttpClient();
```

Comment:
Possibly inline this code into the "if" now? When we had to call Task.Run it was helpful to have the local method, but I don't think it makes sense now.

Reply:
Absolutely, fixed and force-pushed.